io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/TypeSelectionPage.java

```diff
@@ -90,8 +91,8 @@ protected void createContent(Composite page) {
 		GridData layoutData = new GridData();
 		layoutData.widthHint = 200;
 
-		label = new Label(page, SWT.NONE);
-		label.setText("Choose your Type:");
+		label1 = new Label(page, SWT.NONE);
```

Comment:
It would be a bit more readable if you could rename `label1` and `label2` to something appropriate based on what those labels stand for.

Reply:
This is not something I have introduced.